worlds/smw/Rom.py

```diff
@@ -0,0 +1,791 @@
+import Utils
+from Patch import read_rom, APDeltaPatch
+from .Locations import lookup_id_to_name, all_locations
+from .Levels import level_info_dict, full_level_list, submap_level_list, location_id_to_level_id
+from .Names.TextBox import generate_goal_text, title_text_mapping
+
+USHASH = 'cdd3c8c37322978ca8669b34bc89c804'
+ROM_PLAYER_LIMIT = 65535
+
+import hashlib
+import os
+import math
+
+
+ability_rom_data = {
+    0xBC0003: [[0x1F2C, 0x7]], # Run         0x80
+    0xBC0004: [[0x1F2C, 0x6]], # Carry       0x40
+    0xBC0005: [[0x1F2C, 0x2]], # Swim        0x04
+    0xBC0006: [[0x1F2C, 0x3]], # Spin Jump   0x08
+    0xBC0007: [[0x1F2C, 0x5]], # Climb       0x20
+    0xBC0008: [[0x1F2C, 0x1]], # Yoshi       0x02
+    0xBC0009: [[0x1F2C, 0x4]], # P-Switch    0x10
+    #0xBC000A: [[]]
+    0xBC000B: [[0x1F2D, 0x3]], # P-Balloon   0x08
+    0xBC000D: [[0x1F2D, 0x4]], # Super Star  0x10
+}
+
+
+item_rom_data = {
+    0xBC0001: [0x18E4, 0x1], # 1-Up Mushroom
+
+    0xBC0002: [0x1F24, 0x1, 0x1F], # Yoshi Egg
+    0xBC0012: [0x1F26, 0x1, 0x09], # Boss Token
+
+    0xBC000E: [0x1F28, 0x1, 0x1C], # Yellow Switch Palace
+    0xBC000F: [0x1F27, 0x1, 0x1C], # Green Switch Palace
+    0xBC0010: [0x1F2A, 0x1, 0x1C], # Red Switch Palace
+    0xBC0011: [0x1F29, 0x1, 0x1C], # Blue Switch Palace
+
+    0xBC0013: [0x0086, 0x1, 0x0E], # Ice Trap
+    0xBC0014: [0x18BD, 0x7F, 0x18], # Stun Trap
+}
+
+music_rom_data = [
+
+]
+
+level_music_ids = [
+
+]
+
+class LocalRom(object):
```

Comment:
no need to inherit from object in python3
```suggestion
class LocalRom:
```